src/VirtualClient/VirtualClient.Main/CommandBase.cs

```diff
@@ -66,7 +66,7 @@ protected CommandBase()
         /// Parameter defines the content path format/structure using a template to use when uploading content
         /// to target storage resources. When not defined the 'Default' structure is used.
         /// </summary>
-        public string ContentPathPattern { get; set; }
```

Comment:
Needed this incidental change for support of ContentPathTemplate after argument name was modified.